pages/index.html

```diff
@@ -0,0 +1,59 @@
+---
+layout: homepage
+title: CBA is an extension for chrome browser automation
+menu: Homepage
+---
+
+<img src="/images/Manvel_440x280_border.jpg" alt="CBA banner" />
```

Comment:
Detail: let's use more meaning-full image name here. `hero.jpg`, `cba-banner.jpg`.. Guess you can find something more meaningful.